docs/configurations.md

```diff
@@ -101,47 +127,52 @@ The following describes the configurations available for each storage.
 | `scalar.db.jdbc.admin.connection_pool.max_idle`           | Maximum number of connections that can remain idle in the connection pool for admin.                                                                                         | `10`                         |
 | `scalar.db.jdbc.admin.connection_pool.max_total`          | Maximum total number of idle and borrowed connections that can be active at the same time for the connection pool for admin. Use a negative value for no limit.              | `25`                         |
 
-If you use SQLite3 as a JDBC database, you must set `scalar.db.contact_points` as follows.
+{% capture notice--info %}
+**Note**
+
+If you use SQLite3 as a JDBC database, you must set `scalar.db.contact_points` as follows, replacing `YOUR_DB` with the URL of your SQLite3 database:
 
 ```properties
 scalar.db.contact_points=jdbc:sqlite:<YOUR_DB>.sqlite3?busy_timeout=10000
 ```
 
-Unlike other JDBC databases, [SQLite3 does not fully support concurrent access](https://www.sqlite.org/lang_transaction.html).
-To avoid frequent errors caused internally by [`SQLITE_BUSY`](https://www.sqlite.org/rescode.html#busy), we recommend setting a [`busy_timeout`](https://www.sqlite.org/c3ref/busy_timeout.html) parameter.
-
-##### Multi-storage support
+In addition, unlike other JDBC databases, [SQLite3 does not fully support concurrent access](https://www.sqlite.org/lang_transaction.html). To avoid frequent errors caused internally by [`SQLITE_BUSY`](https://www.sqlite.org/rescode.html#busy), we recommend setting a [`busy_timeout`](https://www.sqlite.org/c3ref/busy_timeout.html) parameter.
+{% endcapture %}
 
-ScalarDB supports using multiple storage implementations at the same time.
-You can use multiple storages by specifying `multi-storage` for the `scalar.db.storage` property.
-
-For details about using multiple storages, see [Multi-storage Transactions](multi-storage-transactions.md).
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+</div>
+</div>
 
 ### ScalarDB Server (gRPC)
 
-[ScalarDB Server](scalardb-server.md) is a standalone server that provides a gRPC interface to ScalarDB.
-To interact with ScalarDB Server, you must specify `grpc` for the `scalar.db.transaction_manager` property.
+[ScalarDB Server](scalardb-server.md) is a standalone server that provides a gRPC interface to ScalarDB. To interact with ScalarDB Server, you must specify `grpc` as the value for the `scalar.db.transaction_manager` property.
 
-The following configurations are available for ScalarDB Server:
+The following configurations are available for ScalarDB Server using gRPC transactions:
```

Comment:
Ditto. (in this case, it should probably be `the gRPC transaction manager`.

Reply:
Understood. I've revised this sentence to reflect that.